dev/storage/src/model.tx.json

```diff
@@ -932,7 +932,7 @@
     "_id": "618a9e1165ff379e1b3b8fc7",
     "_class": "core:class:TxCreateDoc",
     "space": "core:space:Tx",
-    "objectId": "618a9e1165ff379e1b3b8fc8",
```

Comment:
Why we need this ones?

Reply:
We need normal ConnectionProviders' IDs to simplify the process of creating candidates from an extension, because we need to know the provider type.

Reply:
ok, sure. I've missed it is human readable name :-)